scripts/test_ports/vcpkg-ci-vsgqt/vcpkg.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "vcpkg-ci-vsgqt",
+  "version-string": "ci",
+  "description": "Port to fvalidate vsgqt CI",
```

```suggestion
  "description": "Port to validate vsgqt CI",
```

Not worth forcing a rebuild for this

---

CI test rebuilds are cheap.

---

```suggestion
  "description": "Port to validate vsgqt in CI",
```